change/@fluentui-eslint-plugin-912d0763-d6cf-4fec-beac-2843c49fc62d.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
## [🕵 FluentUIV0 Open the Visual Regressions report to inspect the affected screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-approval/index.html?organization=uifabric&project=fabricpublic&prId=31988&commitId=b6babc5ccccd616bc45742a35c4a67cc4b839465&env=prod&clientType=FLUENTUI)


  <details><summary><b>List</b> 2 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|List.List Example Selectable.chromium.png|124| Changed|
|List.List Example Selectable.Focuses item.chromium.png|124| Changed|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_v0"/>